cpp/tests/io/parquet_test.cpp

```diff
@@ -4465,6 +4471,132 @@ TEST_F(ParquetWriterTest, CheckColumnOffsetIndexStruct)
   }
 }
 
+TEST_F(ParquetWriterTest, CheckColumnIndexListWithNulls)
+{
+  auto valids  = cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i % 2; });
+  auto valids2 = cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i != 3; });
```

Comment:
You should be able to use our special test iterator here
```suggestion
  auto null_at_3 = cudf::test::iterators::null_at(3);
```